tests/acceptance/features/apiContract/spacesReport.feature

```diff
@@ -0,0 +1,75 @@
+@api @skipOnStable3.0
```

Comment:
I've added `@skipOnStable3.0` tag here but the only step that doesn't work here is the permission check

```feature
 Scenario: check the response of the searched file
    Given user "Alice" has uploaded a file inside space "findData" with content "some content" to "testFile.txt"
    And using new DAV path
    When user "Alice" searches for "testFile.txt" using the WebDAV API
    Then the HTTP status code should be "207"
    And the "REPORT" response to user "Alice" should contain a mountpoint "findData" with these key and value pairs:
      | key                | value               |
      | oc:fileid          | UUIDof:testFile.txt |
      | oc:file-parent     | UUIDof:findData     |
      | oc:name            | testFile.txt        |
      | d:getcontenttype   | text/plain          |
      | oc:permissions     | RDNVW               |
      | d:getcontentlength | 12                  |
```

For instance, the scenario might work if we remove ` | oc:permissions     | RDNVW               |`

```feature
 Scenario: check the response of the searched file
    Given user "Alice" has uploaded a file inside space "findData" with content "some content" to "testFile.txt"
    And using new DAV path
    When user "Alice" searches for "testFile.txt" using the WebDAV API
    Then the HTTP status code should be "207"
    And the "REPORT" response to user "Alice" should contain a mountpoint "findData" with these key and value pairs:
      | key                | value               |
      | oc:fileid          | UUIDof:testFile.txt |
      | oc:file-parent     | UUIDof:findData     |
      | oc:name            | testFile.txt        |
      | d:getcontenttype   | text/plain          |
      | d:getcontentlength | 12                  |
```

We might get into conflicts while merging this stable-3.0 with the master or while cherry-picking the changes in this file from the master. What should we do skip the whole scenario or remove the permission check?

Reply:
Permission does not work means no permission in the response or it gives different permission @SwikritiT ?

Reply:
the permission field is empty

Reply:
This is because the bug related to it https://github.com/owncloud/ocis/issues/6525 was fixed on the master. Don't think it'll be backported

Reply:
yeah bug fix will not be backport in stable branch

Reply:
I can already see the skip tag.

Reply:
https://github.com/owncloud/ocis/pull/6622#issuecomment-1608795262